### PR TITLE
enable GitHub Actions CI

### PR DESCRIPTION
**Pull request by sullis**

### CODE REVIEW

**Comment by slandelle** [comment]
@lsammoc WDYT?

**Review by slandelle — CHANGES_REQUESTED** [review]
If we switch to GH Actions, we should remove the Travis configuration.

**Comment by sullis** [comment]
> If we switch to GH Actions, we should remove the Travis configuration.

Done.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      with:
+        java-version: ${{ matrix.java }}
+    - name: print Java version
+      run: java -version
+    - uses: actions/cache@v2
+      with:
+        path: ~/.m2/repository
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ runner.os }}-maven-
```
Comment (Isammoc) [inline .github/workflows/ci.yml:31]:
```suggestion
        key: maven-${{ runner.os }}-${{ hashFiles('**/pom.xml') }}
        restore-keys: |
          maven-${{ runner.os }}-${{ hashFiles('**/pom.xml') }}
          maven-${{ runner.os }}-
          maven-
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    - name: Build
+      run: mvn --no-transfer-progress -B clean verify
```
Comment (Isammoc) [inline .github/workflows/ci.yml:33]:
```suggestion
      run: mvn --no-transfer-progress -B clean verify
    - name: Clean before caching
      run: find ~/.m2 -name "_remote.repositories" | xargs rm
```

**Review by Isammoc — CHANGES_REQUESTED** [review]

**Comment by Isammoc** [comment]
Thank you, @sullis!

Only minor changes and I think we will be good to go!

@slandelle One task I don't have to do myself ^^ :+1: 
The CLA?

**Comment by slandelle** [comment]
@Isammoc CLA is good, @sullis has already signed it for another PR.

**Comment by sullis** [comment]
Ready for review @slandelle  @Isammoc

**Review by Isammoc — APPROVED** [review]

**Review by slandelle — APPROVED** [review]

**Comment by slandelle** [comment]
@sullis Thanks!

**Pull request merged**